lib_bgp_data/cdn_whitelist/cdn_whitelist.py

```diff
@@ -58,13 +58,15 @@ def _run(self):
             for cdn in self._get_cdns():
                 # Get ASN data
                 response = session.get(api + cdn)
-                # Check for errors
                 response.raise_for_status()
 
-                # Format data for db insertion
-                for line in response.text.split('\n'):
-                    asn = line.split(',')[0].replace('"', '')
-                    whitelist.append([cdn, asn])
+                if response.text == 'error getting result':
+                    pass
+                else:
```

Comment:
if response.text != "error getting result": might be a bit cleaner since you just pass